R/robust_score_test.R

```diff
@@ -88,18 +91,15 @@ robust_score_test <- function(glm_object, call_to_model, param = 1,
                      (c_tilde %*% u_tilde_sum))
 
   } else if (is.na(id)) {
-
-    score_contribution <- function(i, model_fits) { ### output is p x 1
-      matrix(model_fits[i]^2 * (yy[i] - model_fits[i]) * xx[i, ], ncol = 1)
-    }
-
-    fisher_info_contribution <- function(i, model_fits) { ### output is p x p... Di^T Vi Di
-      model_fits[i] * xx[i, ] %*% t(xx[i, ])
-    }
-
-    u_tilde <- sapply(1:nn, score_contribution, model_fits = model0_fits) ### p x n
-
-    aa0 <- Reduce("+", sapply(1:nn, fisher_info_contribution, model_fits = model0_fits, simplify=F)) ### p x p
+    
+    u_tilde <- sapply(1:nn, score_contribution, model_fits = model0_fits, 
+                      family = model1family, link = model1link, xx = xx, yy = yy) ### p x n
+    
+    ##Need to reorder u tilde the way a is reordered??
```

Comment:
this is no longer a question, correct? So we can change to "##Need to reorder u tilde the way a is reordered"